.github/workflows/ci.yml

```
@@ -11,7 +11,7 @@ jobs:
       matrix:
         scala:
           - 2.13.5
```

please keep in sync with `build.sbt`:
```suggestion
          - 2.13.14
```